blueoil/generate_lmnet_config.py

```diff
@@ -127,7 +131,6 @@ def _blueoil_to_lmnet(blueoil_config):
     }
     dataset = {}
 
-
```

Comment:
Why do we remove a line here?

Reply:
My autopep8 removes unnecessary line break such as double empty line in functions.
Is this line necessary?

Reply:
this line does not seem necessary but this diff is not related to the PR and thus should go into another PR.

Reply:
absolutely!